src/components/Ticketbox/components/BackBtn.jsx

```diff
@@ -1,11 +1,11 @@
 import * as React from "react";
 
-const BackBtn = ({ type, switchRenderedType }) => (
+const BackBtn = ({ path, type, switchRenderedType, switchHelpType }) => (
     <input
         type="button"
         value="BACK"
         className={type === 'error' ? "form-error-back-btn" : "form-back-btn"}
```

Comment:
Is there any other type prop other than `error` planned? If not, this can be `error: PropType.bool`